src/test/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItemSpec.scala

```diff
@@ -17,19 +17,23 @@ package nl.knaw.dans.easy.stage.fileitem
 
 import java.io.File
 import java.net.URL
+import java.nio.file.Paths
 
+import nl.knaw.dans.easy.stage.RelationObject
 import nl.knaw.dans.easy.stage.fileitem.EasyStageFileItem._
-import nl.knaw.dans.easy.stage.fileitem.SdoFiles._
+import nl.knaw.dans.easy.stage.fileitem.SdoFiles.readDatastreamFoxml
 import nl.knaw.dans.easy.stage.lib.Fedora
 import nl.knaw.dans.easy.stage.lib.Util.loadXML
-import org.scalatest.{FlatSpec, Matchers}
+import org.apache.commons.io.FileUtils.{ deleteQuietly, readFileToString, write }
+import org.scalatest.{ FlatSpec, Matchers, OneInstancePerTest }
 
 import scala.collection.immutable.HashMap
-import scala.reflect.io.Path
-import scala.util.{Failure, Success, Try}
+import scala.util.{ Failure, Success, Try }
 
-class EasyStageFileItemSpec extends FlatSpec with Matchers {
-  System.setProperty("app.home", "src/main/assembly/dist")
+class EasyStageFileItemSpec extends FlatSpec with Matchers with OneInstancePerTest {
+
+  private val testDir = Paths.get("target/test", getClass.getSimpleName)
+  deleteQuietly(testDir.toFile)
```

Comment:
I guess you should mix in a `BeforeAndAfterEach` trait and put this line in the `override def beforeEach(): Unit` method